### ISSUE TITLE
False positive in detect-non-literal-fs-filename

### ISSUE THREAD

**Issue by Sjord** [returntocorp/sgrep-rules#1467]
[detect-non-literal-fs-filename.yaml](https://github.com/returntocorp/semgrep-rules/blob/develop/javascript/lang/security/audit/detect-non-literal-fs-filename.yaml)

corpus.js:
```
this.open()
```

Run semgrep:
```
$ semgrep --config 'r/javascript.lang.security.audit.detect-non-literal-fs-filename'
using config from https://semgrep.dev/r/javascript.lang.security.audit.detect-non-literal-fs-filename. Visit https://semgrep.dev/registry to see all public rules.
downloading config...
running 1 rules...
corpus.js
severity:warning rule:javascript.lang.security.audit.detect-non-literal-fs-filename.detect-non-literal-fs-filename: A variable is present in the filename argument of fs calls, this might allow an attacker to access anything on your system.
1:this.open()
ran 1 rules on 1 files: 1 findings
```

**Comment by Sjord** [comment]
Ping @inkz

**Comment by inkz** [comment]
Nice catch! thank you :smile: 
PR: https://github.com/returntocorp/semgrep-rules/pull/1469
